ArchUnitNETTests/Fluent/PlantUml/PlantUmlFluentComponentDiagramTests.cs

```diff
@@ -37,6 +37,7 @@ public void ComponentDiagramFromSlicesTest()
             var sliceRule = SliceRuleDefinition.Slices().Matching("ArchUnitNETTests.(*).");
             var uml1 = ComponentDiagram().WithDependenciesFromSlices(sliceRule, Architecture).AsString();
             var uml2 = ComponentDiagram().WithDependenciesFromSlices(sliceRule.GetObjects(Architecture)).AsString();
+            ComponentDiagram().WithDependenciesFromSlices(sliceRule.GetObjects(Architecture)).WriteToFile("../../../Fluent/PlantUml/TestPlantumlDependeciesTest.puml");
```

Comment:
this only tests if the export does not throw an exception, but does not assert if it has the right content. Could you make an assertion for the uml1, uml2 contents instead?